dags/quicksuggest2bq.py

```diff
@@ -0,0 +1,45 @@
+"""
+Runs a Docker image that imports Quicksuggest suggestions
```

Comment:
Looks like this old version of the file still exists.

Reply:
Whoops.